src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleCreationMidEventValidationService.java

```diff
@@ -164,6 +170,14 @@ private void validateSingleCasesState(List<SubmitEvent> submitEvents, List<Strin
 
             }
 
+            if (!isScotland && !isNullOrEmpty(submitEvent.getCaseData().getManagingOffice())
```

Comment:
Do we need this granularity in the validation of Scottish cases? Do we care about managing office for Scotland?

Reply:
Based off the current implementation, no. In Scotland, cases within a multiple can be part of different offices. The work I have done get's it to the same state as ECM. From what you are saying, Glasgow singles should only be added to Glasgow multiples but this has never been the case on ECM

Reply:
I am getting this checked though whether it should be implemented

Reply:
Yeah sorry I didn't read the code carefully enough. Yes it should do the same as ECM does now. You could argue it would be a nicer API if the parameter was caseTypeId rather than isScotland then you can put the logic to determine if it is Scotland in that method rather than ask clients to work it out, but I'm being uber picky there